### ISSUE TITLE
Resize issues with urth-viz-explorer

### ISSUE THREAD

**Issue by peller** [jupyter-incubator/declarativewidgets_explorer#59]
Follow-up to #2, which enables urth-viz-vega to resize based on its container

1. urth-viz-explorer expands to fit the width of the notebook, but in dashboard mode, where the height of the cell changes, the explorer does not expand. This is because the notebook's output divs have no explicit sizing.  We could look up the DOM to the div.cell and set the size to match that.  All other widgets must have the same problem?
1. Also, the collapsible cards may overlap with the cell in dashboard mode.  Should overflow be hidden or scroll?
1. The resize behavior of vega-lite currently assumes width and height are the size of the plot area and does not account for axes or labels.  We've worked around this with a padding fudge factor of 150px.

**Comment by dalogsdon** [comment]
> All other widgets must have the same problem?

Yes, I've seen the problem with other widgets such as the globe.

> Should overflow be hidden or scroll?

Only if there is a use case where the explorer is expected to be used with the query in dashboard view. This also raises the question of whether the expand/JSON buttons should be displayed in dashboard view.
